### DISCUSSION TITLE
editing existing mesh

### DISCUSSION THREAD

**Discussion by ghost** [Jyers/Marlin#497] [Q&A] [answered]
excuse the noob question..
I'm using the 5x5 manual mesh, and I think I set a few of the points a little too low. can I edit the mesh or should I run the levelling again (all 25 points of it)?

**Comment by Jyers** [comment, marked as answer]
At the moment the only option is to run the leveling again, but on the bright side I did add a menu option which restores your previous mesh position while leveling, so you don't have to remeasure all of them, you can just return to previous and adjust.
